apps/publisher/themes/default/js/taxonomy-view-asset.js

```diff
@@ -13,10 +13,13 @@ $(function () {
             if (appliedTaxonomy.length > 0) {
                 for (var key in appliedTaxonomy) {
                     if (appliedTaxonomy.hasOwnProperty(key)) {
+                        var element = appliedTaxonomy[key];
+                        getTaxonomyDisplayName(element);
                         $('.selected-taxonomy-content').append(
                             '<div style="padding: 12px"><span>'
-                            + appliedTaxonomy[key].split('/').join(' > ') + '</span></div>');
+                            + displayValue.join(' > ') + '</span></div>');
                     }
+                    displayValue.length = 0;
```

Comment:
@splinter This is done to clear the existing array. A taxonomy is saved as a path where node id is separated by a forward slash `/`. I have to traverse through each node id and get the display name of each node. I'm saving these display values in `displayValue` array in order to generate the display value of the taxonomy. I'm using the same array to generate multiple taxonomy display names. Hence using `displayValue.length = 0;` to clear the array.